packages/cli/src/codegen/schema.ts

```diff
@@ -306,7 +305,6 @@ Suggestion: add an '!' to the member type of the List, change from '[${baseType}
     const type = tsCodegen.namedType(
       typesCodegen.ascTypeForValue(this._resolveFieldType(gqlType)) as any,
     );
-    // In AssemblyScript, primitives cannot be nullable.
```

Comment:
yeah this will apply to all primitives right now I don't see any harm of this approach. All we are doing is adding guarded logic around code to ensure if it is `null` or not